### DISCUSSION TITLE
StackStorm Exchange Incubator

### DISCUSSION THREAD

**Discussion by lm-ydubler** [StackStorm/st2#5414] [General]
Hi all,
I am a software engineer with LogicMonitor and we recently pushed our [LogicMonitor Pack](https://github.com/StackStorm-Exchange/exchange-incubator/pull/169) to StackStorm Exchange Incubator with the hopes of getting it pushed into StackStorm Exchange. However, the Incubator appears to have a deficit in maintainers and the process of getting the pack pushed to StackStorm Exchange is not clear. So I am reaching out here to get help with this issue...

How do we get our pack approved in StackStorm Exchange Incubator so that it can get onto StackStorm Exchange?

Thanks and best,
Yuri

**Comment by nzlosh** [comment]
Hi Yuri,

I'll take a look in the next couple days and give you feedback.

Regards,
Carlos

**Reply by lm-ydubler** [reply]
Amazing, thank you Carlos. Much appreciated.
